app/controllers/updates_controller.rb

```diff
@@ -0,0 +1,26 @@
+class UpdatesController < ApplicationController
+  def show
+    update_paths = Dir.glob(Rails.root.join('app/views/updates/update_files/*').to_s)
+    @updates = update_file_paths_to_hash(update_paths).sort.reverse.to_h
+  end
+
+  def update_file_paths_to_hash(update_paths)
+    updates_by_date = {}
+    update_paths.each do |update_path|
+        path, name, date_array = process_update_path(update_path)
```

Comment:
indentation here is 4 spaces, I guess we need to set up rubocop to check these too

Reply:
Rubocop automatically did this somehow... but yes I agree